frontend/src/v5/services/api/drawingRevisions.ts

```diff
@@ -60,16 +38,7 @@ export const createRevision = async (
 			Math.round((progressEvent.loaded * 100) / progressEvent.total),
 		),
 	};
-	for (let progress = 0; progress < 100;) {
-		const { promiseToResolve, resolve } = getWaitablePromise();
-		setTimeout(() => {
-			progress = Math.min(progress + Math.round(Math.random() ** 2 * 100), 100);
-			onProgress(progress);
-			resolve();
-		}, Math.random() * 1500);
-		await promiseToResolve;
-	}
-	return delay(500, null);
+	return api.post(`teamspaces/${teamspace}/projects/${projectId}/drawings/${drawingId}/revisions`, body, config);
 };
 
 export const getRevisionFileUrl = (teamspace: string, projectId: string, drawingId: string, revision: string) => (
```

Comment:
Has this been implemented yet? When I try to download a drawing revision I get a `Container not found` error.
![image](https://github.com/user-attachments/assets/6bf40ea5-2d04-45cb-b50c-7ff4dae4f041)